ethers-middleware/tests/transformer.rs

```diff
@@ -133,6 +133,9 @@ async fn ds_proxy_code() {
         .legacy()
         .send()
         .await
+        .unwrap()
+        .await
```

Comment:
same as above. if we produce the receipt, it should be for testing something

Reply:
we are checking the resolution of the tx later so it has to confirm first